src/MoBi.Presentation/Presenter/EditFormulaInPathAndValuesPresenter.cs

```diff
@@ -19,7 +19,7 @@ public interface IEditFormulaInPathAndValuesPresenter : ICommandCollectorPresent
    {
       void Init<TBuildingBlock, TBuilder>(TBuilder formulaOwner, TBuildingBlock buildingBlock, UsingFormulaDecoder formulaDecoder) 
          where TBuilder : PathAndValueEntity, IUsingFormula, IWithDisplayUnit
-         where TBuildingBlock : PathAndValueEntityBuildingBlock<TBuilder>;
+         where TBuildingBlock : class, ILookupBuildingBlock<TBuilder>;
```

Comment:
This is unrelated, but I was suddenly not able to edit InitialConditions building blocks after yesterday.